src/commands/restore/printing_stdout.md

```diff
@@ -29,24 +29,12 @@ contain the file we'd like to print at all (`production.sql`). In this case, you
 can pass rustic the snapshot ID of the snapshot you like to restore:
 
 ```console
-rustic -r /srv/rustic-repo dump 098db9d5 production.sql | mysql
+rustic dump 098db9d5 production.sql | mysql
 ```
 
 Or you can pass rustic a path that should be used for selecting the latest
 snapshot. The path must match the patch printed in the "Directory" column, e.g.:
 
 ```console
-rustic -r /srv/rustic-repo dump --path /production.sql latest production.sql | mysql
-```
-
-It is also possible to `dump` the contents of a whole folder structure to
-stdout. To retain the information about the files and folders rustic will output
-the contents in the tar (default) or zip format:
-
-```console
-rustic -r /srv/rustic-repo dump latest /home/other/work > restore.tar
-```
-
-```console
-rustic -r /srv/rustic-repo dump -a zip latest /home/other/work > restore.zip
+rustic dump --path /production.sql latest production.sql | mysql
 ```
```

Comment:
Why do we want to leave this out?

Reply:
Because rustic doesn't support dumping to zip or tar. I didn't correct all things I found while going through all pages. But I tried to correct things where I just had to delete wrong parts...